docusaurus/docs/user-docs/content-manager/reviewing-content.md

```diff
@@ -3,16 +3,22 @@ title: Reviewing content
 description: Using review workflows to manage content throughout the content creation process.
 ---
 
-# Reviewing content <EnterpriseBadge /> <AlphaBadge />
+# Reviewing content <EnterpriseBadge /> <BetaBadge />
 
-Using the [review workflows](/user-docs/settings/review-workflows) feature, you can manage content throughout the content creation process. In the Content Manager, you can assign content to a specific stage in the review workflow, and move content between stages as needed.
+Using the [review workflows](/user-docs/settings/review-workflows) feature, you can manage content throughout the content creation process. In the Content Manager, you can change the review stage of the content, and keep moving it between stages as needed.
 
-When viewing your content in the Content Manager, you can see the current status of any item in the **Review Stage** column:
+When viewing your content in the Content Manager, you can see the current stage of any item in the **Review Stage** column:
 
 ![Review Stage column](/img/assets/content-manager/review-stage-column.png)
 
-## Assigning content to a review stage
+## Change review stage
 
-As content is created and revised among your team, you can assign content to a specific stage in the review workflow. When [writing or editing content](/user-docs/content-manager/writing-content), you can assign content to a specific stage in the review workflow by clicking on the **Review Stage** dropdown menu in the **Information** section of the content editor:
+As content is created and revised among your team, you can change the review stage of the content to any stage defined in the review workflows (see [Managing review workflows](/user-docs/settings/review-workflows)).
+
+To change the review stage of your content:
+
+1. Access the edit view of your content-type.
+2. In the Information box, click on the *Review stage* drop-down list.
+3. Choose the new review stage of your entry.
```

Comment:
Maybe we should mention that it's automatically saved and doesn't need any user action?